app/src/observables/webworker-messaging/webworker.tsx

```diff
@@ -0,0 +1,111 @@
+import loadWheels from '../../python/load-wheels.py';
+import setupDirectories from '../../python/setup-directories.py';
+
+import { workerMock } from './worker-mock';
+
+import {
+  workerMessengers, IPyodideMessage
+} from './common';
+
+declare let pyodide: any;
+declare let Module: any;
+declare let languagePluginLoader: any;
+
+interface PyodideWorker extends Worker {
+  pyodide: any;
+  Module: any;
+  languagePluginUrl: string;
+  languagePluginLoader: any;
+  document: any;
+}
+
+const receiverMessengers = workerMessengers.receiver
+const senderMessengers = workerMessengers.sender
+const sendReply = workerMessengers.sendReply
+const sendFileTransfer = workerMessengers.sendFileTransfer
+
+let ctx: PyodideWorker
+
+// ctx = self as any;
+// ctx.languagePluginUrl = 'https://pyodide.pymedphys.com/'
+// importScripts('https://pyodide.pymedphys.com/pyodide.js')
+
+ctx = workerMock as any;
+ctx.languagePluginLoader = languagePluginLoader
```

Comment:
For now just pretending like it's in a webworker, when really it's not.

Reply:
Worth commenting?

Reply:
There are some places in the code where I have put comments. It will flow quite obviously what needs to be changed due to Visual Studio's type checking.